app/pages/delete-order/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,115 @@
+import nock from 'nock';
+import { ClientFunction, Selector } from 'testcafe';
+import { extractInnerText } from 'buying-catalogue-library';
+import content from '../manifest.json';
+import { nockAndErrorCheck, setState, authTokenInSession } from '../../../test-utils/uiTestHelper';
+
+const pageUrl = 'http://localhost:1234/order/organisation/order-id/delete-order';
+
+const defaultPageSetup = { withAuth: true };
+const pageSetup = async (setup = defaultPageSetup) => {
+  if (setup.withAuth) {
+    await setState(ClientFunction)('fakeToken', authTokenInSession);
+  }
+};
+
+const getLocation = ClientFunction(() => document.location.href);
+
+fixture('delete-order page - general')
+  .page('http://localhost:1234/order/some-fake-page')
+  .afterEach(async (t) => {
+    await nockAndErrorCheck(nock, t);
+  });
+
+test('when user is not authenticated - should navigate to the identity server login page', async (t) => {
+  nock('http://identity-server')
+    .get('/login')
+    .reply(200);
+
+  await pageSetup({ ...defaultPageSetup, withAuth: false, getRoute: false });
```

Comment:
There currently is no getRoute property in the page setup, could this be removed?

Reply:
We have done it with our next PR, so if possible could it be kept in